app/src/spIP/java/org/dhis2/usescases/sms/DI/ServiceLocator.kt

```diff
@@ -0,0 +1,36 @@
+package org.dhis2.usescases.sms.DI
+
+import org.dhis2.usescases.sms.data.api.ConstantApi
+import org.dhis2.usescases.sms.data.api.ConstantApiImpl
+import org.dhis2.usescases.sms.data.api.OutboundApi
+import org.dhis2.usescases.sms.data.api.OutboundApiImpl
+import org.dhis2.usescases.sms.data.repository.message.MessageTemplateD2Repository
+import org.dhis2.usescases.sms.data.repository.patient.PatientD2Repository
+import org.dhis2.usescases.sms.data.repository.preferred.PreferredLanguageD2Repository
+import org.dhis2.usescases.sms.data.repository.sms.SmsApiRepository
+import org.dhis2.usescases.sms.domain.usecase.SendSmsUseCase
+import org.hisp.dhis.android.core.D2Manager
+
```

Comment:
This class is not used currently but please use it instead of HIlt because is less invasive in the main code to avoid future conflicts and for this functionality it is enough